reporting-app/app/controllers/activities_controller.rb

```diff
@@ -114,6 +115,8 @@ def set_activity_report_application_form
     end
 
     def activity_type_map(type)
+      return nil if type.nil?
+
       case type
       when "work_activity"
         WorkActivity.new
```

Comment:
Not sure if it makes sense to do it here, or after the demo, but with the addition of categories I think we'll probably want to replace `WorkActivity` with `HourlyActivity`.